package.json

```
@@ -13,31 +13,31 @@
-    "@opentelemetry/instrumentation-express": ">= 0.33.3 < 0.34",
-    "@opentelemetry/instrumentation-fastify": ">= 0.32.4 < 0.33",
-    "@opentelemetry/instrumentation-amqplib": ">= 0.36.0 < 0.37",
-    "@opentelemetry/instrumentation-graphql": ">= 0.35.2 < 0.36",
-    "@opentelemetry/instrumentation-http": ">= 0.47.0 < 0.48",
-    "@opentelemetry/instrumentation-ioredis": ">= 0.35.3 < 0.36",
-    "@opentelemetry/instrumentation-knex": ">= 0.32.3 < 0.33",
-    "@opentelemetry/instrumentation-koa": ">= 0.36.2 < 0.37",
-    "@opentelemetry/instrumentation-mongodb": ">= 0.37.0 < 0.38",
-    "@opentelemetry/instrumentation-mongoose": ">= 0.33.4 < 0.34",
-    "@opentelemetry/instrumentation-mysql": ">= 0.34.4 < 0.35",
-    "@opentelemetry/instrumentation-mysql2": ">= 0.34.4 < 0.35",
-    "@opentelemetry/instrumentation-nestjs-core": ">= 0.33.3 < 0.34",
-    "@opentelemetry/instrumentation-pg": ">= 0.36.2 < 0.37",
-    "@opentelemetry/instrumentation-redis": ">= 0.35.4 < 0.36",
-    "@opentelemetry/instrumentation-redis-4": ">= 0.35.5 < 0.36",
-    "@opentelemetry/instrumentation-restify": ">= 0.34.2 < 0.35",
-    "@opentelemetry/sdk-metrics": ">= 1.21.0 < 1.22.0",
-    "@opentelemetry/sdk-trace-base": ">= 1.21.0 < 1.22.0",
-    "@opentelemetry/instrumentation-undici": ">= 0.2.0 < 0.3",
-    "@opentelemetry/sdk-node": ">= 0.48.0 < 0.49",
-    "@prisma/instrumentation": ">= 5.6.0 < 5.10.0",
+    "@opentelemetry/api": ">= 1.8.0 < 1.9.0",
+    "@opentelemetry/core": ">= 1.21.0 < 1.26.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": ">= 0.48.0 < 0.53",
+    "@opentelemetry/instrumentation-amqplib": ">= 0.38.0 < 0.39",
+    "@opentelemetry/instrumentation-express": ">= 0.40.1 < 0.41",
+    "@opentelemetry/instrumentation-fastify": ">= 0.37.0 < 0.38",
+    "@opentelemetry/instrumentation-graphql": ">= 0.41.0 < 0.42",
+    "@opentelemetry/instrumentation-http": ">= 0.52.0 < 0.53",
+    "@opentelemetry/instrumentation-ioredis": ">= 0.41.0 < 0.42",
+    "@opentelemetry/instrumentation-knex": ">= 0.37.0 < 0.38",
+    "@opentelemetry/instrumentation-koa": ">= 0.41.0 < 0.42",
+    "@opentelemetry/instrumentation-mongodb": ">= 0.45.0 < 0.46",
+    "@opentelemetry/instrumentation-mongoose": ">= 0.39.0 < 0.40",
+    "@opentelemetry/instrumentation-mysql": ">= 0.39.0 < 0.40",
+    "@opentelemetry/instrumentation-mysql2": ">= 0.39.0 < 0.40",
+    "@opentelemetry/instrumentation-nestjs-core": ">= 0.38.0 < 0.39",
+    "@opentelemetry/instrumentation-pg": ">= 0.42.0 < 0.43",
+    "@opentelemetry/instrumentation-redis": ">= 0.40.0 < 0.41",
+    "@opentelemetry/instrumentation-redis-4": ">= 0.40.0 < 0.41",
+    "@opentelemetry/instrumentation-restify": ">= 0.39.0 < 0.40",
+    "@opentelemetry/instrumentation-undici": ">= 0.3.0 < 0.5",
+    "@opentelemetry/sdk-metrics": ">= 1.21.0 <= 1.25.0",
+    "@opentelemetry/sdk-node": ">= 0.48.0 <= 0.52",
+    "@opentelemetry/sdk-trace-base": ">= 1.21.0 <= 1.25.0",
+    "@prisma/instrumentation": ">= 5.6.0 < 5.14.0",
```

Since Prisma used to (for this version range) have specific `@opentelemetry/api` versions hardcoded, I think we should tighten the Prisma version range to those that will allow `@opentelemetry/api@1.8.0`.

Going by [this commit's tags](https://github.com/prisma/prisma/commit/0fc0bf1605f0082f3349fca692ccf2d54b7c7aed), that would be `5.11.0` upwards (but we also want to cap it at `< 5.14.0` because of the `@opentelemetry/instrumentation` type issue)

```suggestion
    "@prisma/instrumentation": ">= 5.11.0 < 5.14.0",
```